envi/archs/arm/__init__.py

```diff
@@ -9,6 +9,12 @@
 from envi.archs.arm.disasm import *
 
 class ArmModule(envi.ArchitectureModule):
+    '''following line before function is to help with different architecture
```

Comment:
purely aesthetic, but please place the quotes on their own line:
'''
following...
...
'''